app/src/main/java/chat/rocket/android/layouthelper/ExtRealmModelListAdapter.java

```diff
@@ -37,6 +41,35 @@ protected void notifyRealmModelItemChanged(int position) {
     notifyItemChanged(position + 1);
   }
 
+  public void setLayoutManager(RecyclerView.LayoutManager layoutManager) {
+    this.layoutManager = layoutManager;
+  }
+
+  protected ListUpdateCallback listUpdateCallback = new ListUpdateCallback() {
+    @Override
+    public void onInserted(int position, int count) {
+      notifyItemRangeInserted(position + 1, count);
+      if (layoutManager != null) {
+        layoutManager.scrollToPosition(0);
+      }
```

Comment:
Yep. We could do that. But not adding a new hard `Realm` dependency is the way to go. :)